src/aircraft/FlyByWireA380X.ts

```diff
@@ -0,0 +1,141 @@
+import { AircraftConfigSimType, AircraftFeature, FeatureType } from '../defs'
+import {
+    AircraftConfig,
+    FeatureAddresses,
+    FeatureState,
+    FlapNames,
+    Meta,
+} from '../interface/aircraft'
+
+export default class FlyByWireA380X extends AircraftConfig {
+    meta: Meta = {
+        id: 'flybywire_a380x',
+        name: 'FlyByWire A380X',
+        sim: AircraftConfigSimType.MsFs,
+        enabled: true,
+        priority: 2,
+        author: 'Arthur Parienté <https://github.com/arthurpar06>'
+    }
+
+    features: FeatureAddresses = {
+        [AircraftFeature.BeaconLights]: {
+            LIGHTING_BEACON_0: FeatureType.Int,
+        },
+        [AircraftFeature.LandingLights]: {
+            LIGHTING_LANDING_2: FeatureType.Int,
+        },
+        [AircraftFeature.LogoLights]: {
+            // Lhe logo light isn't available in the FBW A380X at the moment; in the sim, the navigation light currently controls the logo ligh
+            LIGHTING_NAV_0: FeatureType.Int,
+        },
+        [AircraftFeature.NavigationLights]: {
+            LIGHTING_NAV_0: FeatureType.Int,
+        },
+        [AircraftFeature.StrobeLights]: {
+            LIGHTING_STROBE_0: FeatureType.Int,
+        },
+        [AircraftFeature.TaxiLights]: {
+            LIGHTING_LANDING_1: FeatureType.Int,
+        },
+        [AircraftFeature.WingLights]: {
+            LIGHTING_WING_0: FeatureType.Int,
+        },
+        [AircraftFeature.APU]: {
+            A32NX_OVHD_APU_MASTER_SW_PB_IS_ON: FeatureType.Int,
+            A32NX_OVHD_APU_START_SW_PB_IS_ON: FeatureType.Int,
+        },
+        [AircraftFeature.Battery]: {
+            A32NX_OVHD_ELEC_BAT_1_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_2_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_ESS_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_APU_PB_IS_AUTO: FeatureType.Int,
```

Comment:
Why APU ? Batteries (and External Power) should be ok for this.

Reply:
Same as the above discussion about the APU BAT.